### PR TITLE
Gray action buttons on conversation show page are too large

### PR DESCRIPTION
**Pull request by peterp**
One the conversation show page the gray action buttons span half the page. They should span the width of the text only and be aligned with the right hand side of the page.

screen shot 2013-07-10 at 11 30 38 am

### CODE REVIEW

File: go/conversation/templates/conversation/show.html [hunk 1]
```diff
@@ -108,7 +108,7 @@ <h4 class="control-label">
 
     <div class="span5">
         {% if actions %}
-        <div class="sub-actions">
+        <div class="sub-actions pull-right">
```
Comment (justinvdm) [inline go/conversation/templates/conversation/show.html:114]: The ticket description says these actions should be aligned to the right and that the width should span the width of the text. This probably does the "aligning to the right" part, but I think we still need to fix the width part of the ticket description.

Comment (peterp) [inline go/conversation/templates/conversation/show.html:114]: The width is arbitrary as it depends on the width of the text in the buttons. There's no way to effectively match them together given the current structure of the HTML.

Comment (justinvdm) [inline go/conversation/templates/conversation/show.html:114]: Ah ok cool

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**